docs/build-instructions/windows.md

```diff
@@ -33,7 +33,7 @@ cd atom
 script/build # Creates application in the `Program Files` directory
```

Comment:
I noticed you didn't change this instance of `script/build`. Just wondering why that's so.